src/components/Skeleton/Skeleton.tsx

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import styled, { keyframes } from "styled-components";
+import { SkeletonProps } from "./types";
+
+const waves = keyframes`
+   from {
+        left: -150px;
+    }
+    to   {
+        left: 100%;
+    }
+`;
+
+const pulse = keyframes`
+  0% {
+    opacity: 1;
+  }
+  50% {
+    opacity: 0.4;
+  }
+  100% {
+    opacity: 1;
+  }
+`;
+
+const Root = styled.div<{ variant: string; width: number | undefined; height: number | undefined }>`
+  min-height: 20px;
+  display: block;
+  background-color: #bdc2c4;
+  border-radius: 4px;
```

Comment:
`radii.small` from theme

Reply:
better define this as a ternary : 

```
border-radius: ${({ variant, theme }) => (variant === 'circle ? theme,radii.circle : theme,radii.small)};
```

and remove the line 33

Reply:
Excellent @RabbitDoge  I didn't know the theme variables my fault. Fixing it!